plugins/python/src/main/kotlin/io/paddle/plugin/python/extensions/Environment.kt

```diff
@@ -47,9 +47,10 @@ class Environment(val project: PaddleProject, val venv: VenvDir) : Hashable {
         override fun create(project: PaddleProject): Environment {
             val config = object : ConfigurationView("environment", project.config) {
                 val venv by string("path", default = ".venv")
+                val noIndex by lazy { get<String>("noIndex")?.toBoolean() ?: false }
```

Comment:
I would parse the boolean value, but `by boolean` is buggy and doesn't work as expected: It throws a cast exception. I suppose it's because it's trying to convert string to boolean with `as Boolean`